docs/console/docs/01-01-console.md

```diff
@@ -11,4 +11,14 @@ Use the following CRs to modify the Console UI:
 
 - The MicroFrontend custom resource allows you to plug in micro front-ends for a specific Namespace.
 - The ClusterMicroFrontend custom resource allows you to plug in micro front-ends for the entire Cluster.
-- The BackendModule custom resource allows you to enable UI API Layer modules.
\ No newline at end of file
+- The BackendModule custom resource allows you to enable UI API Layer modules.
+  
+
+#### Security guidelines for `MicroFrontend` and `ClusterMicroFrontend` resources:
+
+We highly recommend to include the following configuration to your `MicroFrontend` or `ClusterMicroFrontend` web server config file:
```

Comment:
```suggestion
For security reasons, follow the listed guidelines when you add the MicroFrontend or ClusterMicroFrontend configuration in the web server file:
```
I would say the following points are rather guidelines than the configuration itself, that's why I changed the introductory line - for the user to follow the guidelines when defining the configuration that goes into the web server file.

Reply:
`Web server file` sounds weird IMHO, and also this is unrelated to the resource configuration itself, but to the web server serving the microfrontend. This would be my counterproposal, let me know if it is fine for you:
_For security reasons, follow the listed guidelines when you configure the web server for the MicroFrontend or ClusterMicroFrontend_

Reply:
done